tests/e2e/specs/navigation.js

```diff
@@ -1,182 +1,6 @@
-
-describe('Testing Navigation', () => {
-  // const baseURL = 'http://localhost:8080/';
-  // const homePath = '/';
-  const eventsPath = '/upcoming-events';
-  const checkoutPath = '/checkout';
-  const profilePath = '/profile';
-  const registrationsPath = '/my-events';
-
-  beforeEach(() => {
-    cy.login();
+describe('Verify that we can build the code and see the login page', () => {
+  it('shows login', () => {
+    cy.visit('/');
+    cy.contains('p', 'The Sajni Center: A space of hope and healing. We are excited to invite you and your family to view the wide selection of group interactive therapies that we offer.');
```

Comment:
Probably should just search for "The Sajni Center" since the other text may reasonably change